cob_script_server/src/simple_script_server/simple_script_server.py

```diff
@@ -697,7 +697,7 @@ def compose_color(self,component_name,parameter_name):
 			full_parameter_name = self.ns_global_prefix + "/" + component_name + "/" + parameter_name
 			if not rospy.has_param(full_parameter_name):
 				rospy.logerr("parameter %s does not exist on ROS Parameter Server, aborting...",full_parameter_name)
-				return 2
+				return 2, None
```

Comment:
this fix only applies to this specific function because it is a helper function which can rise errors, but where the handle doesn't exist.
All other functions do either not rise errors or have access to '''ah '''